### DISCUSSION TITLE
Remote Octoprint?

### DISCUSSION THREAD

**Discussion by AdamTheLawyer** [UnchartedBull/OctoDash#1943] [Q&A] [answered]
Can the OctoDash interface communicate with an OctoPrint instance on another machine?

I installed OctoDash/OctoPrint on an old Wintel tablet (on top of a Linux install), which worked OK, but there's only one USB port available (and I need to connect the printer, keyboard, webcam, mouse...) and it's not really an ideal platform for OctoPrint

I'd rather run OctoPrint on a Pi and just run OctoDash on a tablet to control it remotely. (Or maybe use the tablet for OctoDash and the webcam server to free up the PI from doing it). I tried changing the server address to the Pi, but it didn't seem to want to connect, which made me wonder if there are hooks that require OctoDash and OctoPrint on the same machine.

Ideally, I'd use an Android tablet for OctoDash, but I saw from a previous thread that making an Android version would be prohibitively difficult.

Is there a relatively easy way to build a windows version? Testing, at least, would be easier if I could just have OctoDash on my desktop.

Thank you for all your hard work--it's a fantastic tool.

**Comment by jneilliii** [comment, marked as answer]
Yes, it is possible to run OctoDash remotely from the octoprint server/pi as long as the two devices are on the same network and can communicate with each other. The OctoDash Companion plugin however would not work in this situation as it expects them both to be running on the same device.

**Reply by AdamTheLawyer** [reply]
:) Excellent. I'll have to try it.

Octodash Companion? What is ... <googles> ...

:) This looks like fun.

**Reply by jneilliii** [reply]
It's a plugin I wrote to help people add custom action buttons and upload themes easily to OctoDash.
